fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -898,10 +899,15 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
 
           // Update CRZ DEST predictions
           const destPred = this.props.fmcService.master?.guidanceController?.vnavDriver?.getDestinationPrediction();
+          let destEta = '--:--';
           if (destPred?.secondsFromPresent !== undefined) {
-            this.destEta.set(new TimeHHMMFormat().format(destPred.secondsFromPresent / 60)[0] ?? '--:--');
-            this.destEfob.set(this.props.fmcService.master?.fmgc.getDestEFOB(true).toFixed(1) ?? '--.-');
+            destEta = getEtaUtcOrFromPresent(
+              destPred.secondsFromPresent,
+              this.activeFlightPhase.get() == FmgcFlightPhase.Preflight,
+            );
           }
+          this.destEta.set(destEta);
+          this.destEfob.set(this.props.fmcService.master?.fmgc.getDestEFOB(true).toFixed(1) ?? '--.-');
```

Comment:
```
'--.-'
```

Just checking that this is intentional vs --.--, else LGTM

Reply:
Yes that's fine as it is the fuel on board but I updated it to be '---.-' now here & other pages as they seem to use 3 dashes on the A380 for fuel figures. 
![image](https://github.com/user-attachments/assets/f5df7ea6-86ac-478a-9522-e48f51c1d76a)